LICENSE.md

```diff
@@ -0,0 +1,7 @@
+## Copyright © 2019 <Victor Shchenkh>
```

Comment:
Here's a template which seems to be recognisable by GitHub: https://github.com/severen/LICENSE.md/blob/master/MIT.md